tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+import responses
+
+
+@pytest.fixture
+def mocked_responses():
```

Comment:
Maybe this could be set to `autouse=True`? I don't know offhand whether that disrupts some specifictests, but I imagine we probably just want this on all the time.

Reply:
You're right, this should probably be on all the time to prevent unexpected I/O. I'll try that out and push an update or report back.

Reply:
Done. Thanks for this suggestion!